src/Components/Form.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const Form = (props) => {
+  const { newBookFunc } = props;
+  // const [title, setTitle] = useState('');
+  // const [author, setAuthor] = useState('');
```

Comment:
It's good practice to remove all the commented code from your project. Kindly remove the commented code. 👍🏻